src/Raven.Studio/typescript/models/database/tasks/ongoingTaskSqlEtlEditModel.ts

```diff
@@ -127,15 +127,13 @@ class ongoingTaskSqlEtlEditModel extends ongoingTaskEditModel {
             Disabled: this.taskState() === "Disabled",
             MentorNode: this.manualChooseMentor() ? this.mentorNode() : undefined,
             PinToMentorNode: this.pinMentorNode(),
-            FactoryName: "System.Data.SqlClient",
```

Comment:
don't we need to put here `Microsoft.Data.SqlClient` ?

Reply:
no